extensions/industry/healthcare/fhir/fieldgroups/coverage.schema.json

```diff
@@ -23,233 +23,238 @@
-                  "$ref":"https://ns.fhir.org/datatypes/simplequantity"
-               },
-               "xdm:valueMoney":{
-                  "title":"Value Money",
-                  "description":"The amount due from the patient for the cost category.",
-                  "$ref":"https://ns.fhir.org/datatypes/money"
-               },
-               "xdm:exception":{
-                  "title":"Exception",
-                  "description":"A suite of codes indicating exceptions or reductions to patient costs and their effective periods.",
-                  "type":"array",
-                  "items":{
-                     "type":"object",
-                     "properties":{
-                        "xdm:type":{
-                           "title":"Type",
-                           "description":"The code for the specific exception..",
-                           "$ref":"https://ns.fhir.org/datatypes/codeableconcept"
-                        },
-                        "xdm:period":{
-                           "title":"Period",
-                           "description":"The timeframe the exception is in force.",
-                           "$ref":"https://ns.fhir.org/datatypes/period"
+                     "xdm:exception":{
+                        "title":"Exception",
+                        "description":"A suite of codes indicating exceptions or reductions to patient costs and their effective periods.",
+                        "type":"array",
+                        "items":{
+                           "type":"object",
+                           "properties":{
+                              "xdm:type":{
+                                 "title":"Type",
+                                 "description":"The code for the specific exception..",
+                                 "$ref":"https://ns.fhir.org/datatypes/codeableconcept"
+                              },
+                              "xdm:period":{
+                                 "title":"Period",
+                                 "description":"The timeframe the exception is in force.",
+                                 "$ref":"https://ns.fhir.org/datatypes/period"
+                              }
+                           }
                         }
                      }
+                  },
+                  "xdm:subrogation":{
+                     "title":"Subrogation",
+                     "description":"When 'subrogation=true' this insurance instance has been included not for adjudication but to provide insurers with the details to recover costs.",
+                     "type":"boolean"
+                  },
+                  "xdm:contract":{
+                     "title":"Contract Details",
+                     "description":"The policy(s) which constitute this insurance coverage.",
+                     "description":"The insurer assigned ID for the Subscriber.",
```

Comment:
There is a duplicate "description" that needs to be removed

Reply:
done